src/app.scss

```diff
@@ -30,13 +30,17 @@
 }
 
 .files-empty-state,
-.files-view-stack > .pf-v5-c-card {
+.files-view-stack > .pf-v5-c-card,
+.files-view-stack > .files-footer-info {
```

Comment:
I consolidated all the `grid-column: content` directives into one multi-element selector.